src/nameserver/name_server_impl.cc

```diff
@@ -2335,17 +2326,18 @@ int NameServerImpl::SetPartitionInfo(TableInfo& table_info) {
     return 0;
 }
 
-int NameServerImpl::CreateTableOnTablet(std::shared_ptr<::openmldb::nameserver::TableInfo> table_info, bool is_leader,
-                                        std::map<uint32_t, std::vector<std::string>>& endpoint_map, uint64_t term) {
+int NameServerImpl::CreateTableOnTablet(const std::shared_ptr<::openmldb::nameserver::TableInfo>& table_info,
+                                        bool is_leader, std::map<uint32_t, std::vector<std::string>>& endpoint_map,
+                                        uint64_t term) {
     ::openmldb::type::CompressType compress_type = ::openmldb::type::CompressType::kNoCompress;
     if (table_info->compress_type() == ::openmldb::type::kSnappy) {
         compress_type = ::openmldb::type::CompressType::kSnappy;
     }
     ::openmldb::api::TableMeta table_meta;
     table_meta.set_db(table_info->db());
     table_meta.set_name(table_info->name());
-    table_meta.set_tid(table_info->tid());
-    table_meta.set_seg_cnt(table_info->seg_cnt());
+    table_meta.set_tid(static_cast<::google::protobuf::int32>(table_info->tid()));
```

Comment:
I have seen tid is invalid(<=0) while printing the tid which is more than 0 in MacOS. Not sure if it is related to this.

Reply:
Probably yes, chenjing and zekai also met this error. Maybe error rate is high in MacOS.